apps/coscrad-frontend/src/utils/generic-components/float-spacer.tsx

```diff
@@ -1 +1,7 @@
-export const FloatSpacerDiv = (): JSX.Element => <div className="spacer">&nbsp;</div>;
+import { styled } from '@mui/material';
+
+const StyledDiv = styled('div')({
```

Comment:
We may want to come up with a naming convention for these styling components that use `styled`

Reply:
this is a super important pattern that we need to put serious thought to. let's plan a design session for this.

Reply:
Sounds good